src/main/java/org/opentest4j/ValueWrapper.java

```diff
@@ -68,7 +71,7 @@ public static ValueWrapper create(Object value) {
 	 *                             want to rely on {@link Object#toString()} will fallback
 	 *                             to default behavior if null
 	 * @return a wrapper for the supplied value
-	 * @since 1.2.0
+	 * @since 1.2
 	 */
 	public static ValueWrapper create(Object value, String stringRepresentation) {
 		return (value == null ? nullValueWrapper : new ValueWrapper(value, stringRepresentation));
```

Comment:
I think we should also check for `value instanceof ValueWrapper` down here. If it the same `stringRepresentation`, we can simply return it. Otherwise, if it has an ephemeral value, we can unwrap it and return a new `ValueWrapper`. Thoughts?